### PR TITLE
[FlashScript] add other Shelly devices to detection function.

### PR DESCRIPTION
**Pull request by andyblac**
This will stop crash if they are on local network.

ie
```
Scanning for Shelly devices...

Exception in thread zeroconf-ServiceBrowser__http._tcp.local._9240336:
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.8/3.8.6/Frameworks/Python.framework/Versions/3.8/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.8/site-packages/zeroconf/__init__.py", line 1721, in run
    self._service_state_changed.fire(
  File "/usr/local/lib/python3.8/site-packages/zeroconf/__init__.py", line 1479, in fire
    h(**kwargs)
  File "/usr/local/lib/python3.8/site-packages/zeroconf/__init__.py", line 1577, in on_change
    listener.add_service(*args)
  File "./flash_shelly.py", line 146, in add_service
    info = get_info(device)
  File "./flash_shelly.py", line 126, in get_info
    info['model'] = shelly_model(info['type'])
  File "./flash_shelly.py", line 175, in shelly_model
    return(options[type])
KeyError: 'SHBTN-1'
```

### CODE REVIEW

**Comment by rojer** [comment]
make the code not crash on unknown devices. we shouldn't be handling devices firmware doesn't support and script should not start crashing when new devices come out.

**Comment by andyblac** [comment]
ok done.

File: tools/flash_shelly.py [hunk 1]
```diff
@@ -172,7 +188,8 @@ def shelly_model(type):
              'dimmer2' : 'ShellyDimmer2',
              'rgbw2' : 'ShellyRGBW2',
   }
-  return(options[type])
+  value = options[type] if type in options else type
```
Comment (rojer) [inline tools/flash_shelly.py:191]: value = options.get(type, type)

Comment (andyblac) [reply tools/flash_shelly.py]: done, thanks, did not know that one.

**Pull request merged**